pkg/notes/notes.go

```diff
@@ -722,7 +731,7 @@ func (g *Gatherer) prsFromCommit(commit *gogithub.RepositoryCommit) (
 ) {
 	githubPRs, err := g.prsForCommitFromMessage(*commit.Commit.Message)
 	if err != nil {
-		logrus.Infof("No PR found for commit %s: %v", commit.GetSHA(), err)
+		logrus.Debugf("No PR found for commit %s: %v", commit.GetSHA(), err)
```

Comment:
@puerco since we're talking about release logs -- is this intentional to be in `Info` level? because I found it to be noisy when running release notes locally, but I wonder if this is a desired feature?

alternatively, if it _is_ desired, would it make sense to run with `Debug` level for releases?

Reply:
TBH I'm not sure when or how often this case can happen. I don't remember seeing it much :thinking: